Code/Framework/AzManipulatorTestFramework/Source/ImmediateModeActionDispatcher.cpp

```diff
@@ -75,15 +75,19 @@ namespace AzManipulatorTestFramework
     {
         ToggleOn(GetMouseInteractionEvent()->m_mouseInteraction.m_mouseButtons.m_mouseButtons, MouseButton::Left);
         GetMouseInteractionEvent()->m_mouseEvent = AzToolsFramework::ViewportInteraction::MouseEvent::Down;
-        m_manipulatorViewportInteraction.GetManipulatorManager().ConsumeMouseInteractionEvent(*m_event);
+        auto mouseEvent = *GetMouseInteractionEvent();
+        mouseEvent.m_mouseInteraction.m_mouseButtons.m_mouseButtons = static_cast<AZ::u32>(MouseButton::Left);
```

Comment:
Should we prefer `aznumeric_cast` on these?

Reply:
Ah yes good point I can update that :+1: